docker/README.md

```diff
@@ -92,3 +92,36 @@ Built from [`dexterous-hand:kinetic`](#dexterous-handkinetic). Adds support for
 ### dexterous-hand:kinetic-nvidia-ide
 
 Built from [`dexterous-hand:kinetic`](#dexterous-handkinetic). Adds support for Nvidia GPUs. [Dockerfile](shadow-hand/kinetic/nvidia_ide/Dockerfile)
+
+# Oneliner
```

Comment:
It would be nice add more details what this one-liner is. Since these days we have so many one-liners :)